pokemongo_bot/cell_workers/utils.py

```diff
@@ -144,3 +145,57 @@ def print_yellow(message):
 
 def print_red(message):
     print(u'\033[91m' + message.decode('utf-8') + '\033[0m')
+
+def nickname_pokemon(bot,pokemon):
+    """This requies a pokemon object containing all the standard fields: id, ivs, cp, etc"""
+    template = bot.config.nickname_template.lower()
+    if template == "{name}":
```

Comment:
Forgive me as I don't know what the pokemon variable looks like at this point, but are you saying ```pokemon.get('nickname')``` exists at this point of the code?